### PR TITLE
Aleth ui fixes

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

**Comment by winsvega** [comment]
SendDialog.ui  could be resized by height
address input field now does not match the length of eth address "0xb337d2ae9219aa0709b3e343df5714db9bfa26f3" (example)

when entering passphrase for the first time in alethone, input dialog suddenly change it's size

File: libaleth/SendDialog.ui [hunk 1]
```diff
@@ -2,6 +2,32 @@
 <ui version="4.0">
  <class>SendDialog</class>
  <widget class="QDialog" name="SendDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>223</width>
```
Comment (chriseth) [inline libaleth/SendDialog.ui:9]: I think it's not a good idea to fix the pixel size of anything. What if the font size is different?

Comment (winsvega) [inline libaleth/SendDialog.ui:9]: it's adjusted by qtFontMetric

Comment (LianaHus) [inline libaleth/SendDialog.ui:9]: I didn't

Comment (LianaHus) [inline libaleth/SendDialog.ui:9]: @winsvega SendDialog can't be resized. at least on my version. are you sure you check the latest one? or you meant AlethOn? Do you think we should fix the height for AlethOne?

Comment (LianaHus) [inline libaleth/SendDialog.ui:9]: you are right there is a small gap. but the question for AlethOne is still valid :)

Comment (LianaHus) [inline libaleth/SendDialog.ui:9]: I would fix everything but log part. the white part at the bottom.

File: libaleth/SendDialog.cpp [hunk 2]
```diff
@@ -34,17 +34,23 @@ SendDialog::SendDialog(QWidget* _parent, AlethFace* _aleth):
...
 	adjustDialogWidth();
 }
 
 void SendDialog::adjustDialogWidth()
 {
-	QString str("0x42e6723a0c884e922240e56d7b618bec96f35800"); //just some ethereum address
-	QFont defaultFont("Sans Serif", 16);
-	QFontMetrics fm(defaultFont);
+	QString str("0xd57b6c60f48f4187edf735f627400bf03e5c7f02"); //just some ethereum address
+	QFont defaultFont("Sans Serif");
```
Comment (winsvega) [inline libaleth/SendDialog.cpp]: defaultFont not used

**Pull request merged**